src/components/Checkbox.ts

```diff
@@ -0,0 +1,20 @@
+import { Option, Checkboxes } from '@slack/types';
```

Comment:
I would not export 3rd party types directly (though I think I already have done that). this will make versioning harder and potentially cause conflicts when consuming this. eg if consuming project already depends on another version of slack types.

also will make it harder to implement other platforms in the future.

probably better to create our own types for stuff we export from now on (at least for our component props)

Reply:
makes sense. i moved the `SelectOption` from the select component to a shared location, since this was already an internal version of the slack `Option` type, and use here